skimage/external/test_tifffile.py

```diff
@@ -17,15 +17,15 @@
 np.random.seed(0)
 
 
-@skipif(si is None)
+@pytest.mark.skipif(si is None, reason="skimage not installed")
```

Comment:
Can we add a utility function to utils instead, and then apply that here?  I'm just thinking about our next transition from pytest to future_test_utility.